### PR TITLE
Component - use namespaces for sidebar, submenu, and toolbar

### PR DESCRIPTION
**Pull request by travisrisner**
This PR uses swaps the sidebar, submenu and toolbar to utilize namespaces.

These are things I find myself swapping each time when using this boilerplate.

### CODE REVIEW

File: component/administrator/components/com_foo/helpers/foo.php [hunk 1]
```diff
@@ -31,6 +32,6 @@ class FooHelper
 	 */
 	public function addSubmenu($vName)
 	{
-		JHtmlSidebar::addEntry(Text::_('COM_FOO'), 'index.php?option=com_foo&view=foo', $vName == 'foo');
+		HTML::_('sidebar.addEntry', Text::_('COM_FOO'), 'index.php?option=com_foo&view=foo', $submenu == 'foo');
```
Comment (roland-d) [inline component/administrator/components/com_foo/helpers/foo.php]: @travisrisner I think `HTML::` should be `HTMLHelper`

Comment (travisrisner) [reply component/administrator/components/com_foo/helpers/foo.php]: Thank you so much! This should be resolved now.

File: component/administrator/components/com_foo/helpers/foo.php [hunk 2]
```diff
@@ -31,6 +32,6 @@ class FooHelper
 	 */
 	public function addSubmenu($vName)
 	{
-		JHtmlSidebar::addEntry(Text::_('COM_FOO'), 'index.php?option=com_foo&view=foo', $vName == 'foo');
+		HTMLHelper::_('sidebar.addEntry', Text::_('COM_FOO'), 'index.php?option=com_foo&view=foo', $submenu == 'foo');
```
Comment (roland-d) [inline component/administrator/components/com_foo/helpers/foo.php]: @travisrisner There is another issue :) The `$submenu` should be `$vName` as that is the variable passed in.

Comment (travisrisner) [reply component/administrator/components/com_foo/helpers/foo.php]: Thanks for the catch. This should be resolved now.

**Pull request merged**